frontend/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -68,16 +71,21 @@ export function* collectUndoStates(): Saga<void> {
     const curTracing = yield* select(state => enforceSkeletonTracing(state.tracing));
     if (userAction) {
       if (curTracing !== prevTracing) {
+        if (!shallSkipPreviousState(userAction, previousAction)) {
```

Comment:
I'd negate the semantics of this helper method and name it `shouldAddToUndoStack`, because it seems clearer and easier to parse to me (since there is no double negation).